app/src/main/java/ch/epfl/favo/user/User.java

```diff
@@ -64,34 +64,38 @@ public String getNotificationId() {
     return notificationId;
   }
 
+  void setNotificationId(String notificationId) {
+    this.notificationId = notificationId;
+  }
+
   public LocalDate getBirthDate() {
     return birthDate;
   }
 
-  public Location getLocation() { return location; }
+  public Location getLocation() {
+    return location;
+  }
 
```

Comment:
any reason why you moved this function defintion up? not a big deal but was wondering because we had getters and setters both listed in order of fields

Reply:
This was done automatically by the reformat tool of Android studio, I'll change it back as it was before

Reply:
oh i must;ve forgot to reformat this one when i added it, sorry about that

Reply:
This is the way google format does it.